CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * **Improve hierarchy detection in pptx documents** List items, and other slide text are properly nested under the slide title. This will enable better chunking of pptx documents.
 * **Refactor of the ingest cli workflow** The refactored approach uses a dynamically set pipeline with a snapshot along each step to save progress and accommodate continuation from a snapshot if an error occurs. This also allows the pipeline to dynamically assign any number of steps to modify the partitioned content before it gets written to a destination.
 * **Applies `max_characters=<n>` argument to all element types in `add_chunking_strategy` decorator** Previously this argument was only utilized in chunking Table elements and now applies to all partitioned elements if `add_chunking_strategy` decorator is utilized, further preparing the elements for downstream processing.
+* **Add retry strategy for source connectors** Dynamic retry strategy added with exponential backoff added to notion source connector.
```

Comment:
right now this only adds to Notion, correct? headline here might be misleading?

Reply:
The enhancement was the addition of the retry strategy itself. The comment associated with the bullet points states it was only applied to the notion connector.

Reply:
right, added suggestion as to how to clarify